### ISSUE TITLE
Does PGB support post-training quantization?

### ISSUE THREAD

**Issue by yazdavar** [facebookresearch/PyTorch-BigGraph#156]
I am wondering if PGB support Post-training quantization for instance what we we have for fasttext: https://flavioclesio.com/2019/03/22/post-training-quantization-in-fasttext-or-how-to-shrink-your-fasttext-model-in-90/

**Comment by lw** [comment]
No, that's not something we ship, you'd have to do it yourself. Depending on what you're looking for, FAISS may also help, as I think I remember it has some compression/quantization techniques.

**Comment by lw** [comment]
If you're looking for nearest neighbors take a look at FAISS, I think it has tools to deal with mmapped files (so that only some chunks at a time are loaded from disk to memory) and with compression.
